modules/ROOT/pages/runtime-installation-task.adoc

```diff
@@ -23,103 +23,168 @@ The Mule 4 binary files are located under *Downloads* > *Mule Runtime* on the Do
 +
 . Set an environment variable called `MULE_HOME` for the `mule` directory inside your extracted folder.
 +
-Example for version 4.1.1 in the `Downloads` directory:
+Example for version 4.1.5 in the `Downloads` directory:
 +
-On Windows:
+* On Windows environments:
 +
 [source,console]
 ----
-$ env:MULE_HOME=C:\Downloads\mule-enterprise-standalone-4.1.1\
+$ env:MULE_HOME=C:\Downloads\mule-enterprise-standalone-4.1.5\
 ----
 +
-On Linux/Unix environments:
+* On Linux/Unix environments:
 +
 [source,console]
 ----
-$ export MULE_HOME=~/Downloads/mule-enterprise-standalone-4.1.1/
+$ export MULE_HOME=~/Downloads/mule-enterprise-standalone-4.1.5/
 ----
 
-== Run the Mule Runtime
+== Running Mule
 
-You can test if the Mule Runtime runs in your system without errors by running the following commands:
+You can test if Mule runs in your system without errors by running the following commands:
 
 * On Windows environments:
 +
 [source,powershell]
 ----
 $ $MULE_HOME\bin\mule.bat
 ----
-+
+
 * On Linux/Unix environments:
 +
 [source,console]
 ----
 $ $MULE_HOME/bin/mule
 ----
 
-== Start the Mule Service
+These commands run Mule in foreground mode, and the startup script displays information on the terminal’s standard output. You can’t issue further commands on the terminal as long as Mule is running.
 
-You can run the Mule runtime standalone by starting the Mule service inside `$MULE_HOME/bin/` directory:
+To stop Mule, press `CTRL-C` in the terminal in which the script is running.
+
+== Starting Mule as a Windows Service
+
+To run Mule as a Windows service, you need to install it first by running:
 
-* On Windows environments:
-+
-To run the Mule service, you need to install it first by running:
-+
 [source,powershell]
 ----
 $ $MULE_HOME\bin\mule.bat install
 ----
-+
-Once installed, you can run the service:
-+
+
+Once installed, you can run Mule as a service:
+
 [source,powershell]
 ----
 $ $MULE_HOME\bin\mule.bat start
 ----
+
+[NOTE]
+====
+When Windows restarts, the Mule service stops the same way as using the `mule stop` command.
+The only situation where this could be different is if Windows kills the process due to a timeout.
+====
+
+=== Adding Properties to wrapper.conf File After Installation
+
+Properties that are defined after the installation of Mule as a Windows service
```

Comment:
> Properties that are defined after the installation of Mule as a Windows service in the `wrapper.conf` file can be overridden by properties generated in the `wrapper.additional.conf` file.

The point of this sentence is confusing due to the usage of the passive voice. Maybe something like this would be easier to understand at a first glance:

> After the installation of Mule as a Windows service, you can override the properties defined in the `wrapper.conf` file by declaring new properties using the `wrapper.additional.conf` file.